### ISSUE TITLE
Web - Search Page - Search Box Missing Label Text and Submit Button

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Comply-Chain#88]
There is no text for the LABEL for the search box. 

There is an ID for the INPUT field but no FOR on the LABEL.

Search - https://dolcontentdev.opadev.dol.gov/ilab/complychain/build/search

**Comment by KyleLivingstonDOL** [comment]
ZTI, I'm not sure I understand, but if you do, can you please rectify?

**Comment by Abychalil** [comment]
Colton Daily added a comment - 15/Jul/19 11:48 PM (#250 in JIRA)
Not clear on the ask. Is a label needed for the search box? The search box already contains "Search" text.

**Comment by jfc3-dol** [comment]
Yes, a LABEL is required, so the user knows the use of the  INPUT field. Plus, not all screen readers read the place holder text that goes inside the INPUT field along with once the user tabs or taps into the field they don't know what it is. They wonder was that my full name, first name, last name, etc. if they weren't paying attention to what needs to be entered.

**Comment by jfc3-dol** [comment]
The LABEL FOR and the INPUT field's ID are now there and both match.

So now I'm going to close this issue.
